Dashboard/Views/ScrutineeringWindow.axaml

```diff
@@ -0,0 +1,129 @@
+<UserControl xmlns="https://github.com/avaloniaui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:vm="using:Dashboard.ViewModels"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:utils="clr-namespace:Dashboard.Utils"
+             mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
+             x:Class="Dashboard.Views.ScrutineeringWindow"
+             x:DataType="vm:ScrutineeringViewModel"
+             Foreground="Black"
+             Background="#565656">
+    <Design.DataContext>
+        <!-- This only sets the DataContext for the previewer in an IDE,
+             to set the actual DataContext for runtime, set the DataContext property in code (look at App.axaml.cs) -->
+        <vm:ScrutineeringViewModel />
+    </Design.DataContext>
+    
+    <UserControl.Resources>
+        <!-- This allows the conditional rendering of the "DV Data" header -->
+        <utils:MeasurmentVisibility x:Key="MeasurmentVisibility" />
+    </UserControl.Resources>
+
+    <Panel>
+        <Grid>
+            <Grid.RowDefinitions>
+                <!-- Top 10% of the screen -->
+                <RowDefinition Height="0.1*" />
+                <!-- Auto sizes to the height of the content -->
+                <RowDefinition Height="Auto" />
+                <!-- Middle 70% of the screen (the carousel) -->
+                <RowDefinition Height="0.7*" />
+                <RowDefinition Height="Auto" />
+            </Grid.RowDefinitions>
+            <Grid.ColumnDefinitions>
+                <!-- 20% left margin -->
+                <ColumnDefinition Width="0.2*" />
+                <!-- Carousel takes 60% of horizontal space -->
+                <ColumnDefinition Width="0.6*" />
+                <!-- 20% right margin -->
+                <ColumnDefinition Width="0.2*" />
+            </Grid.ColumnDefinitions>
+            <StackPanel Grid.Row="0" Grid.Column="1" ZIndex="1" Margin="10 5">
```

Comment:
This panel (and the 2 text blocks below) end up overlapping the content when the screen is resized

Reply:
![image](https://github.com/user-attachments/assets/6c80e535-a6ea-4856-997b-4b8ab5c1e629)

Reply:
this doesnt occur for me

Reply:
6d30670d3dd57630e69139a3b1a94ba94b62176e